File: w24w09javalotto/src/main/java/kr/ac/kumoh/s20210463/w24w09javalotto/controller/LottoController.java
package kr.ac.kumoh.s20210463.w24w09javalotto.controller;

import kr.ac.kumoh.s20210463.w24w09javalotto.model.LottoNumber;
import kr.ac.kumoh.s20210463.w24w09javalotto.service.LottoService;
import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class LottoController {
    private final LottoService lottoService;

    public LottoController(LottoService lottoService) {
        this.lottoService = lottoService;
    }

    @GetMapping("/lotto/numbers")
    public String generateNumbers(Model model) {
        LottoNumber lottoNumber = lottoService.getLuckyNumbers();
        model.addAttribute("numbers", lottoNumber.getNumbers());
        return "lotto";
    }
}

File: w24w07lotto/src/main/kotlin/kr/ac/kumoh/s20210463/w24w07lotto/contoller/LottoController.kt
package kr.ac.kumoh.s20210463.w24w07lotto.contoller

import kr.ac.kumoh.s20210463.w24w07lotto.model.LottoNumber
import kr.ac.kumoh.s20210463.w24w07lotto.service.LottoService
import org.springframework.stereotype.Controller
import org.springframework.ui.Model
import org.springframework.web.bind.annotation.GetMapping

@Controller
class LottoController(val service: LottoService) {
    @GetMapping("/lotto/numbers")
    fun generateNumbers(model: Model): String {
        val lottoNumber: LottoNumber = service.getLuckyNumbers()
        model.addAttribute("numbers", lottoNumber.numbers)
        return "lotto"
    }
}